themes/tdr/login/messages/messages_en.properties

```diff
@@ -1,7 +1,7 @@
 # Standard Keycloak template message TDR overrides
 backToApplication=Sign in to your account
 betaBanner=BETA
-betaBannerInfo=This is a new service - your feedback will help us to improve it. Please\u0020
+betaBannerInfo=This is a new service — your feedback will help us to improve it. Please\u0020
```

Comment:
I think the n-dash needs to be encoded as its not displaying correctly:

![Screenshot from 2021-06-07 07-59-12](https://user-images.githubusercontent.com/51711388/120973545-fcfd9180-c766-11eb-92b8-d164afa964bb.png)

Should be something like this I think: `betaBannerInfo=This is a new service &ndash; your feedback will help us to improve it. Please\u0020`

See: https://www.w3.org/wiki/Common_HTML_entities_used_for_typography

Reply:
Thanks Tom :)